### PR TITLE
You tube service

### PR DESCRIPTION
**Pull request by KmBarnett**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Adds the youtube API call to the back end
## Related Tickets & Documents
#10 
## Added tests?

- [x] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [ ] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

giphy

### CODE REVIEW

File: app/labor/video_suggester.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      relevant_terms = raw_json.find_all {|concept| concept[:relevance].round(3) >= 0.85}
+
+      relevant_terms.map{|term| term[:text]}[0..2]
+    else
+      ['Beginner', 'Software', 'Development']
+    end
+  end
+
+  def self.get_youtube_videos(tags, terms)
+    # require 'pry'; binding.pry
```
Comment (kathleen-crowe) [inline app/labor/video_suggester.rb]: need to remove this pry

File: app/labor/video_suggester.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  def self.get_watson_concepts(article)
+    raw_json = WatsonService.get_keywords(article)
+
+    if !raw_json.nil? && raw_json.include?(:concepts) && raw_json[:concepts] != []
+      relevant_terms = raw_json.find_all {|concept| concept[:relevance].round(3) >= 0.85}
+
+      relevant_terms.map{|term| term[:text]}[0..2]
+    else
+      ['Beginner', 'Software', 'Development']
```
Comment (caachz) [inline app/labor/video_suggester.rb:15]: Glad we added the sad path

File: spec/labor/video_suggester_spec.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  scenario "returns list of terms" do
+
+    json_response = File.read('spec/fixtures/files/watson_response.json')
+
+    response = stub_request(:post, "https://api.us-south.natural-language-understanding.watson.cloud.ibm.com/v1/analyze?features=concepts&url=https://dev.to/vinaybommana7/django-
+      mysql-how-to-port-your-web-application-from-sqlite-to-mysql-3jnl&version=2019-07-12").to_return(status: 200, body: json_response)
+
+      url = 'https://dev.to/vinaybommana7/django-mysql-how-to-port-your-web-application-from-sqlite-to-mysql-3jnl'
+
+      # create_list(:article, 4, user: user, featured: true, tags: ["discuss"])
```
Comment (kathleen-crowe) [inline spec/labor/video_suggester_spec.rb]: need to remove this comment as well

File: app/labor/video_suggester.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    end
+  end
+
+  def self.get_youtube_videos(tags, terms)
+    # require 'pry'; binding.pry
+    search_terms = "#{tags.split(',').join(' | ')} + #{terms.join(' | ')}"
```
Comment (caachz) [inline app/labor/video_suggester.rb:20]: Good formatting before feeding the info to youtube

**Review by kathleen-crowe — CHANGES_REQUESTED** [review]
Everything looks good except removing those commented out pieces of code

**Pull request merged**